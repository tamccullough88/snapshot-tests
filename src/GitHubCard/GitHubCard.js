import React from "react"
import Card from "react-bootstrap/Card"
import photo from './Photo-with-dog.jpg'

function GitHubCard() {
    return (
        <div className="card">
            <Card style={{ width: "18rem" }}>
                {/* My Github does not have a photo, so I am using another photo */}
                <Card.Img variant="top" src={photo} />

                <Card.Body>
                    <Card.Title>Photo of me with a dog</Card.Title>
                    <Card.Text>
                        I am a software dev student learning about testing.
                    </Card.Text>
                </Card.Body>
            </Card>
        </div>
    );
}

export default GitHubCard