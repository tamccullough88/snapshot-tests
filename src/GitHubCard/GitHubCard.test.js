import renderer from "react-test-renderer";
import GitHubCard from "./GitHubCard";


test('renders a snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})