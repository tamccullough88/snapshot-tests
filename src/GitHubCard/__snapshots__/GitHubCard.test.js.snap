// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="Photo-with-dog.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Photo of me with a dog
      </div>
      <p
        className="card-text"
      >
        I am a software dev student learning about testing.
      </p>
    </div>
  </div>
</div>
`;
